bucket/f3d.json

```diff
@@ -1,14 +1,20 @@
 {
-    "version": "1.3.1",
+    "version": "2.0.0",
     "description": "F3D, a fast and minimalist 3D viewer",
-    "homepage": "https://f3d-app.github.io/f3d/",
+    "homepage": "https://f3d.app",
     "license": "BSD-3-Clause",
     "architecture": {
         "64bit": {
-            "url": "https://github.com/f3d-app/f3d/releases/download/v1.3.1/f3d-1.3.1-win64.zip",
-            "hash": "e2ffa54bbab102e55b6cd0ee9d4d2e92cac8be1093e90dade61d17bd42f2af46"
+            "url": "https://github.com/f3d-app/f3d/releases/download/v2.0.0/F3D-2.0.0-Windows-x86_64-raytracing.zip",
+            "hash": "md5:103644766af57381b93481fcbe7eff3e",
+            "extract_dir": "F3D-2.0.0-Windows-x86_64-raytracing"
         }
     },
+    "pre_install": [
+        "if (!(Test-Path \"$persist_dir\\config.json\")) {",
+        "    Copy-Item \"$dir\\share\\f3d\\configs\\config.d\\10_global.json\" \"$dir\\share\\f3d\\configs\\config.json\" -Force -ErrorAction SilentlyContinue",
```

Comment:
Copy `...\config.d\10_global.json` to `...\configs\config.json` is equivalent to default behavior. the whole `$dir\share\f3d\configs` should be persisted

> Windows: %APPDATA%\f3d\(config.json,.d), [install_dir]\share\f3d\configs\(config.json,.d)

Reply:
Fixed. I had assumed copying one of the example configs was needed.